ledger-tool/src/bigtable.rs

```diff
@@ -261,34 +274,13 @@ async fn shreds(
     // shreds being signed with the "dummy" keyapir can still be inserted and
     // later read/replayed/etc
     let keypair = keypair_from_seed(&[0; 64])?;
-    let ShredConfig {
-        shred_version,
-        num_hashes_per_tick,
-        num_ticks_per_slot,
-        allow_mock_poh,
-    } = shred_config;
 
     for slot in slots.iter() {
         let block = bigtable.get_confirmed_block(*slot).await?;
-        let entry_summaries = match bigtable.get_entries(*slot).await {
-            Ok(summaries) => Some(summaries),
-            Err(err) => {
-                let err_msg = format!("Failed to get PoH entries for {slot}: {err}");
-
-                if allow_mock_poh {
-                    warn!("{err_msg}. Will create mock PoH entries instead.");
-                } else {
-                    return Err(format!(
-                        "{err_msg}. Try passing --allow-mock-poh to allow \
-                        creation of shreds with mocked PoH entries"
-                    ))?;
-                }
-                None
-            }
-        };
```

Comment:
Heh, we should have matched on Ok/Err in the first place

Reply:
Haha agreed - this became quite obvious to me when adding the code that matched on `ShredPohGenerationMode`